src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1738,7 +1738,7 @@ void MachUEPNode::emit(CodeBuffer& cbuf, PhaseRegAlloc* ra_) const
   C2_MacroAssembler _masm(&cbuf);
 
   Label skip;
-  __ cmp_klass(j_rarg0, t1, t0, skip);
+  __ cmp_klass(j_rarg0, t1, t0, t2 /* as a tmp */, skip);
```

Comment:
You might also want to add comment for 't0' here: /* as a temp */

Reply:
The other two are done, and the comments like this are renamed to a more friendly `/* call-clobbered t2 as a tmp */`.

t0 is always used freely as a tmp register, I guess it is not necessary to add a comment for t0? It seems only the newly-added t2 here matters.